app/_how-tos/use-ai-rag-injector-plugin.md

```diff
@@ -0,0 +1,580 @@
+---
+title: Use AI RAG Injector plugin
+content_type: how_to
+related_resources:
+  - text: AI Gateway
+    url: /ai-gateway/
+  - text: AI RAG Injector
+    url: /plugins/ai-rag-injector/
+  - text: AI Proxy Advanced
+    url: /plugins/ai-proxy-advanced/
+
+description: Configure the AI RAG Injector plugin.
+
+products:
+  - gateway
+  - ai-gateway
+
+works_on:
+  - on-prem
+
+min_version:
+  gateway: '3.6'
+
+plugins:
+  - ai-proxy-advanced
+
+entities:
+  - service
+  - route
+  - plugin
+
+tags:
+  - ai
+  - openai
+
+tldr:
+  q: How do I use the AI RAG Injector plugin to safeguard that my company chatbot responds with relevant questions regarding compliance policies?
+  a: Use the AI RAG Injector plugin to integrate your company’s compliance policy documents as retrieval-augmented knowledge. Configure the plugin to inject context from these documents into chatbot prompts, ensuring it can generate relevant, accurate compliance-related questions dynamically during conversations.
+
+
+tools:
+  - deck
+
+prereqs:
+  inline:
+    - title: OpenAI
+      include_content: prereqs/openai
+      icon_url: /assets/icons/openai.svg
+    - title: Redis stack
+      content: |
+          To complete this task, you must have a Redis stack configured in your environment. Check [Redis website](https://redis.io/docs/latest/) to learn more.
+      icon_url: /assets/icons/redis.svg
+    - title: PgVector (optionally)
+      content: |
+        Test
+      icon_url: /assets/icons/database.svg
+    - title: Langchain splitters
+      content: |
+        To complete this tutorial, you'll need **Python (version 3.7 or later)** and `pip` installed on your machine. You can verify it by running:
+
+        ```bash
+        python --version
+        pip --version
+         ```
+
+        Once that's set up, install the required packages by running the following command in your terminal:
+        ```
+        pip install langchain langchain_text_splitters requests
```

Comment:
```suggestion
        python3 -m pip install langchain langchain_text_splitters requests
```
The pip install commands weren't working for me even though I added pip to my path.